src/smt/theory_lra.cpp

```diff
@@ -341,7 +341,6 @@ class theory_lra::imp {
         }
         app_ref cnst(a.mk_numeral(rational(c), is_int), m);
         TRACE("arith", tout << "add " << cnst << "\n";);
-        enode* e = mk_enode(cnst);
```

Comment:
I need the call to mk_enode(cnst), but the return value can be ignored.

Reply:
Okay, I will improve this fix then. Thanks for the review!